pyproject.toml

```diff
@@ -44,6 +44,7 @@ pytest = "7.4.0"
 coverage = "7.2.7"
 pytest-mock = "3.11.1"
 pytest-timeout = "2.2.0"
+minio = "^7.2.7"
```

Comment:
Why is minio defined as test dependency?
AFAIK it is needed just on Ray cluster, right? For that it should be enough to have it declared in `mnist_pip_requirements.txt`

Reply:
Ohh right I missed to remove it... Previously I was using minio test dependancy in support.py to download datasets but as this is merged in mnist.py itself so this will not be needed..
I will remove it right away :+1: 
Thank you @sutaakar for pointing this out!